ImmersionThemePlugin.inc.php

```diff
@@ -40,6 +40,7 @@ public function init() {
 		$this->addOption('sectionDescriptionSetting', 'radio', array(
 			'label' => 'plugins.themes.immersion.options.sectionDescription.label',
 			'description' => 'plugins.themes.immersion.options.sectionDescription.description',
+			'default' => '#000',
```

Comment:
Is this the correct default value? It should probably be `disable` or `enable`, yeah?

Reply:
I meant to use #000 as the default announcement and description sections on the homepage colours but had another brain fart 🤦‍♀

Reply:
Fixed and force-pushed